rmgpy/rmg/reactors.py

```diff
@@ -56,3 +56,97 @@
+        arrs = [to_rms(arr) for arr in obj.arrhenius]
+        return rms.PDepArrhenius(Ps, arrs)
+    elif isinstance(obj, MultiArrhenius):
+        arrs = [to_rms(arr) for arr in obj.arrhenius]
+        return rms.MultiArrhenius(arrs)
+    elif isinstance(obj, MultiPDepArrhenius):
+        parrs = [to_rms(parr) for parr in obj.arrhenius]
+        return rms.MultiPdepArrhenius(parrs)
+    elif isinstance(obj, Chebyshev):
+        Tmin = obj.Tmin.value_si
+        Tmax = obj.Tmax.value_si
+        Pmin = obj.Pmin.value_si
+        Pmax = obj.Pmax.value_si
+        coeffs = obj.coeffs.value_si.tolist()
+        return rms.Chebyshev(coeffs, Tmin, Tmax, Pmin, Pmax)
+    elif isinstance(obj, ThirdBody):
+        arr = to_rms(obj.arrheniusLow)
+        efficiencies = {spc.label: float(val) for spc,  val in obj.efficiencies.items() if val != 1}
+        return rms.ThirdBody(arr, nameefficiencies=efficiencies)
+    elif isinstance(obj, Lindemann):
+        arrlow = to_rms(obj.arrheniusLow)
+        arrhigh = to_rms(obj.arrheniusHigh)
+        efficiencies = {spc.label: float(val) for spc,  val in obj.efficiencies.items() if val != 1}
+        return rms.Lindemann(arrhigh, arrlow, nameefficiencies=efficiencies)
+    elif isinstance(obj, Troe):
+        arrlow = to_rms(obj.arrheniusLow)
+        arrhigh = to_rms(obj.arrheniusHigh)
+        efficiencies = {spc.label: float(val) for spc,  val in obj.efficiencies.items() if val != 1}
+        alpha = obj.alpha
+        T1 = obj._T1.value_si if obj._T1 is not None else 0.0
+        T2 = obj._T2.value_si if obj._T2 is not None else 0.0
+        T3 = obj._T3.value_si if obj._T3 is not None else 0.0
+        return rms.Troe(arrhigh, arrlow, alpha, T3, T1, T2, nameefficiencies=efficiencies)
+    elif isinstance(obj, StickingCoefficient):
+        if obj._T0.value_si != 1:
+            A = obj._A.value_si / (obj._T0.value_si) ** obj._n.value_si
+        else:
+            A = obj._A.value_si
+        n = obj._n.value_si
+        Ea = obj._Ea.value_si
+        return rms.StickingCoefficient(A, n, Ea)
+    elif isinstance(obj, NASAPolynomial):
+        return rms.NASApolynomial(obj.coeffs, obj.Tmin.value_si, obj.Tmax.value_si)
+    elif isinstance(obj, NASA):
+        return rms.NASA([to_rms(poly) for poly in obj.polynomials], rms.EmptyThermoUncertainty())
+    elif isinstance(obj, Species):
+        atomnums = dict()
+        for atm in obj.molecule[0].atoms:
+            if atomnums.get(atm.element.symbol):
+                atomnums[atm.element.symbol] += 1
+            else:
+                atomnums[atm.element.symbol] = 1
+        bondnum = len(obj.molecule[0].get_all_edges())
+        rad = rms.getspeciesradius(atomnums, bondnum)
+        diff = rms.StokesDiffusivity(rad)
+        th = obj.get_thermo_data()
+        thermo = to_rms(th)
+        return rms.Species(obj.label, obj.index, "", "", "", thermo, atomnums, bondnum, diff, rad, obj.molecule[0].multiplicity-1, obj.molecular_weight.value_si)
```

Comment:
You mean like as a postprocessing thing? If someone wanted to add that in I don't think it would be very problematic to add the adjlist in here which I suspect is the fastest (also it works even if there are surface species), but for the moment if we don't have a use for it I think it's better to avoid spending time generating them and memory storing them.

Reply:
I agree!